<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="theLayout">
   <item>
   <widget class="QWidget" name="itemBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>500</height>
    </rect>
   </property>
    <layout class="QVBoxLayout" name="itemLayout">
     <item>
      <layout class="QHBoxLayout" name="layoutLabel">
       <item>
        <widget class="QLabel" name="labelLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Label</string>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editLabel"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutConnection">
       <item>
        <widget class="QLabel" name="labelConnection">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Connection</string>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioLocal">
         <property name="text">
          <string>Local</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioRemote">
         <property name="text">
          <string>Remote</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
     <widget class="QWidget" name="widgetAddress">
      <layout class="QHBoxLayout" name="layoutAddress">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="labelAddress">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Address</string>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editAddress" />
       </item>
       <item>
        <widget class="QLabel" name="portLabel">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="editPort">
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>6667</number>
         </property>
         <property name="margin" stdset="0">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
      </widget>
     </item>
     <item>
     <widget class="QWidget" name="locWidgetChannels">
      <layout class="QHBoxLayout" name="layoutChannels">
       <item>
        <widget class="QLabel" name="labelChannels">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Channels</string>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editChannels">
         <property name="placeholderText">
          <string>(optional, channels to join to, multiple split by spaces</string>
         </property>
        </widget>
       </item>
      </layout>
      </widget>
     </item>
     <item>
     <widget class="QWidget" name="locWidgetNick">
      <layout class="QHBoxLayout" name="layoutNick">
       <item>
        <widget class="QLabel" name="labelNick">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Nick</string>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editNick">
         <property name="placeholderText">
          <string>(optional, user name will be used if remains empty)</string>
         </property>
        </widget>
       </item>
      </layout>
      </widget>
     </item>
     <item>
     <widget class="QWidget" name="remWidgetUser">
      <layout class="QHBoxLayout" name="layoutUser">
       <item>
        <widget class="QLabel" name="labelUser">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Nick</string>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editUser">
         <property name="placeholderText">
          <string>(require, username which user is identified with)</string>
         </property>
        </widget>
       </item>
      </layout>
      </widget>
     </item>
     <item>
     <widget class="QWidget" name="remWidgetPass">
      <layout class="QHBoxLayout" name="layoutPass">
       <item>
        <widget class="QLabel" name="labelPass">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editPass">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>(require, password for user account)</string>
         </property>
        </widget>
       </item>
      </layout>
      </widget>
     </item>
    </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>341</width>
       <height>35</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
